File: mo-common/mo-common-oss/src/main/java/ink/ckx/mo/common/oss/query/DeleteFileQuery.java
package ink.ckx.mo.common.oss.query;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/14
 */
@Data
@Schema(description = "删除文件Query")
public class DeleteFileQuery {

    @NotBlank(message = "key不能为空")
    @Schema(description = "key")
    private String key;
}

File: mo-common/mo-common-oss/src/main/kotlin/ink/ckx/mo/common/oss/query/DeleteFileQuery.kt
package ink.ckx.mo.common.oss.query

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/14
 */
@Schema(description = "删除文件Query")
data class DeleteFileQuery(

    @field:NotBlank(message = "key不能为空")
    @Schema(description = "key")
    var key: String? = null
)